src/patternfly/utilities/Text/examples/Text.md

```diff
@@ -0,0 +1,118 @@
+---
+id: Text
+section: utilities
+---
+
+import './Text.css'
+
+## Examples
+
+### Sizing
+
+```hbs
+{{#> text text--modifier="pf-u-font-size-xs"}}
+  Extra small text
+{{/text}}
+{{#> text text--modifier="pf-u-font-size-sm"}}
+  Small text
+{{/text}}
+{{#> text text--modifier="pf-u-font-size-md"}}
+  Medium text
+{{/text}}
+{{#> text text--modifier="pf-u-font-size-lg"}}
+  Large text
+{{/text}}
+{{#> text text--modifier="pf-u-font-size-xl"}}
+  Extra large text
+{{/text}}
+{{#> text text--modifier="pf-u-font-size-2xl"}}
+  2xl text
+{{/text}}
+{{#> text text--modifier="pf-u-font-size-3xl"}}
+  3xl text
+{{/text}}
+{{#> text text--modifier="pf-u-font-size-4xl"}}
+  4xl text
+{{/text}}
+```
+
+### Font Weight
+
+```hbs
+{{#> text text--modifier="pf-u-font-weight-light"}}
+  Light
+{{/text}}
+{{#> text text--modifier="pf-u-font-weight-normal"}}
+  Normal
+{{/text}}
+{{#> text text--modifier="pf-u-font-weight-bold"}}
+  Bold
+{{/text}}
+```
+
+### Font Color
+
+```hbs
+{{#> text text--modifier="pf-u-color-100"}}
+  Font color 100
+{{/text}}
+{{#> text text--modifier="pf-u-color-200"}}
+  Font color 200
+{{/text}}
+{{#> text text--modifier="pf-u-color-300"}}
+  Font color 300
+{{/text}}
+{{#> text text--modifier="pf-u-color-400"}}
+  Font color 400
+{{/text}}
+```
+
+### Font Alignment
```

Comment:
Maybe rename this to align with the table's description of these modifiers? https://pf4.patternfly.org/documentation/core/components/table#controlling-text

We call it "text modifiers" in the data list but that seems a little generic of a label to use here https://pf4.patternfly.org/documentation/core/components/datalist#text-modifiers

```suggestion
### Controlling text
```